desmume/src/frontend/posix/shared/ctrlssdl.cpp

```diff
@@ -513,7 +513,7 @@ process_ctrls_event( SDL_Event& event,
                 break;
```

Comment:
why is it ok for lshift to break, but not rightshift ? shouldn't both fall-through (or goto) the default case ?

Reply:
in which case the simplest solution would be to factor the default case to outside the switch, so it's always executed.

Reply:
the keyboard config goes through desmume_config_read_file so it should indeed be possible to set left shift as an action button too.